birdhouse/config/magpie/magpie.ini.template

```diff
@@ -49,6 +49,37 @@ ziggurat_foundations.session_provider_callable = magpie.models:get_session_calla
 github.client_id=${GITHUB_CLIENT_ID}
 github.client_secret=${GITHUB_CLIENT_SECRET}
 
+# temporary token definition (defaults below if omitted)
+# note: token here refers to uuids employed in temporary URL endpoints, not security auth tokens
+magpie.token_expire = 86400
```

Comment:
This change should be logged also. 
It will auto logged out users after 1day if they did not access the platform (token refresh).

Reply:
Isn't the auto logging out of users related to security auth tokens?
I thought the `magpie.token_expire` variable concerned temporary tokens, which are related to pending requests (for user registration or group T&C confirmation), and that it wouldn't affect the login of a user.

Reply:
You're right! I mixed with `magpie.cookie_expire`. I've been looking at docker-auth tokens too long recently 😅